openwrt/wifi-disconnect-low-signal/wifi-disconnect-low-signal.lua

```diff
@@ -121,6 +121,9 @@ end
 
 function STA:update(info)
    self.signal = info.signal
+   if self.signal > 0 then
+      self.signal = -120
```

Comment:
while this looks ok, I wonder if this has to do with fitting in 8 bits and values under -128 would go to positive. In this case would be nicer to do `self.signal = -128 - self.signal`

could you check if this makes sense?

Reply:
Yes that might be the case. If so, then this makes a lot more sense than replace with random value.

Reply:
nice, could you squash the commits in a single one, since the second is changing the first?

Then I'll merge